Set vote chart font size to zero

RaceProUK opened the pull request:
To stop it spilling into the bar for people who have weird fonts installed on their systems :)

RaceProUK committed:
Set vote chart font size to zero

To stop it spilling into the bar for people who have weird fonts installed on their systems :)

AccaliaDeElementia commented:
Thank you for your Pull request. As the primary maintainer of this plugin is currently in the middle of an active game I unfortunately will have to defer accepting this PR until after the game so that we can perform proper testing of this issue and to prevent accidentally spoiling the game in progress by leaking information.

your pull request is important to us and will be dealt with in the order it was received (just, y'know..... after the game.)

;-)

---

As a comment, I would tend to say that the open vote bars clearly intend to display the text as they explicitly show the number of votes to lynch a specific target, so the correct solution would be to alter the positioning of the text, rather than the size, but without discussion with the primary maintainer of this plugin i am not comfortable making that determination solo.

yamikuronue commented:
Why would you set it to 0? If vote bars are set to "open", the number should show up on the bar. If it's set to 0, wont' it not show up, or be super tiny at best?

RaceProUK committed:
Reset font size and move text down

Because it's an open game, and people want to see the votes.

And because the hedgoh r teh dum :blush:

RaceProUK commented:
Cool, it's automatically pulling through a less dumb fix :)
